ui/apps/platform/src/routePaths.ts

```diff
@@ -336,14 +336,12 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
     },
     // Risk Acceptance must precede generic Vulnerability Management in Body and so here for consistency.
     'vulnerability-management/risk-acceptance': {
-        featureFlagRequirements: allDisabled(['ROX_VULN_MGMT_2_GA']),
```

Comment:
Is this deleted line still needed?

Reply:
Yes, and no.

I'm going to do this change in a follow up that both removes the Risk Acceptance link and removes the deferral workflow CVE table from the VM 1.0 image page. Those will both be dependent on `ROX_VULN_MGMT_UNIFIED_DEFERRAL` instead of the `ROX_VULN_MGMT_2_GA` feature flag, since if for some reason we don't go GA but _do_ release the new deferral flow, we won't have to worry about backwards compatibility. This way both of those conceptually related changes are both on the same, separate PR.